research/src/pages/checkbox.proposal.mdx

```diff
@@ -98,11 +93,11 @@ The `<checkbox>` control is primarily leveraged to indicate a binary state. For
 
 #### Keyboard
 
-Keyboard users can operate the `<checkbox>` as follows:
+Keyboard users can operate the `<oui-checkbox>` as follows:
 
-- `<checkbox>` is eligible for document focus.
-- `<checkbox>` is eligible for `autofocus` when the `autofocus` attribute is present on the host.
-- `<checkbox>` can be the target of `click` events (when not `disabled`) either through mouse click, user tap, or keyboard `space` key press.
+- `<oui-checkbox>` is eligible for document focus.
+- `<oui-checkbox>` is eligible for `autofocus` when the `autofocus` attribute is present on the host.
```

Comment:
Small question about terminology. Since I do not actively involved and read everything I can miss something.
I in general understand sentence and it's spirit. But what confusing me is before I have mapping between word `host` and tag `<host>` in the example. And I clearly see that <host> tag indicate just structure of the component, but not the how I would write component. Right now `<oui-checkbox>` too close to actual way how to write component in the HTML, and I do not immidiately associate that HTML with actual anatomy of the component.

Reply:
> And I clearly see that tag indicate just structure of the component, but not the > how I would write component.

Can you expound, I'm not sure how to address your feedback to provide clarity

Reply:
Naming discussion: #184

Reply:
@una I get this if we change `host` to be that of `oui-host`. That said, he is saying that this isn't how he would write the component in HTML which is what is confusing me.